.pre-commit-config.yaml

```diff
@@ -15,6 +15,7 @@ repos:
       - id: markdownlint-fix
         args:
           - --dot
+          - --disable=MD033
```

Comment:
Oh also I globally disabled this. Because I think we _should_ be using HTML when it improves clarity.

Reply:
Fair enough. But can we add it in a file? This would only disable in `pre-commit`. https://github.com/paddyroddy/paddyroddy/blob/ebffbbfae8e36cf65424597dac9bde70978e1a3f/.markdownlint.yaml#L1-L3

Reply:
For me, adding it to pre-commit-config is the lesser evil to yet another tool config yaml file in the top-level directory.

Reply:
It's a dotfile, I don't see the harm. Plus my IDE would be screaming at me

Reply:
Ah soz, I didn't see it was already there 🤦. Moved.

Reply:
Oh yeah line length